pkg/operation/common/types.go

```diff
@@ -18,6 +18,9 @@ const (
 	// VPNTunnel dictates that VPN is used as a tunnel between seed and shoot networks.
 	VPNTunnel string = "vpn-shoot"
 
+	// KubeAPIServerUsersPrefix is a constant for a prefix used for the users KubeAPIServer instance using a TLS certificate from a trusted origin.
```

Comment:
```suggestion
	// KubeAPIServerPrefix is a constant for a prefix used for the KubeAPIServer instance which uses a TLS certificate from a trusted origin.
```
There is no users and operators distinction for the kube-apiserver ingress If I'm not mistaken.